src/object/task/taskterraform.cpp

```diff
@@ -54,7 +54,7 @@ CTaskTerraform::CTaskTerraform(COldObject* object) : CForegroundTask(object)
     m_lastParticle = 0.0f;
     m_soundChannel = -1;
 
-    assert(m_object->Implements(ObjectInterfaceType::Powered));
+    assert(m_object->GetNumSlots() == 1);
```

Comment:
Should it be `assert(HasPowerCellSlot(m_object))` ?

Reply:
I think so